projects/tinkerbell/sandbox/docker-compose.yml

```diff
@@ -198,41 +131,25 @@ services:
       PACKET_VERSION: ${PACKET_VERSION:-ignored}
       ROLLBAR_TOKEN: ${ROLLBAR_TOKEN:-ignored}
       ROLLBAR_DISABLE: ${ROLLBAR_DISABLE:-1}
-      MIRROR_HOST: ${TINKERBELL_HOST_IP:-127.0.0.1}:8080
-      OSIE_PATH_OVERRIDE: https://projectbuildpipeline-857-pipelineoutputartifactsb-10ajmk30khe3f.s3.us-west-2.amazonaws.com/projects/tinkerbell/hook/latest/54bae38625828b69259eccb3357e13c269437e28 
+      OSIE_PATH_OVERRIDE: https://tinkerbell-storage-for-eksa.s3.us-west-2.amazonaws.com/misc/osie/current
       DNS_SERVERS: 8.8.8.8
       PUBLIC_IP: $TINKERBELL_HOST_IP
       BOOTP_BIND: $TINKERBELL_HOST_IP:67
       HTTP_BIND: $TINKERBELL_HOST_IP:80
       SYSLOG_BIND: $TINKERBELL_HOST_IP:514
       TFTP_BIND: $TINKERBELL_HOST_IP:69
-      DOCKER_REGISTRY: $TINKERBELL_HOST_IP 
-      REGISTRY_USERNAME: ${TINKERBELL_REGISTRY_USERNAME:-admin}
-      REGISTRY_PASSWORD: ${TINKERBELL_REGISTRY_PASSWORD:-Admin1234}
+      DOCKER_REGISTRY: public.ecr.aws/l0g8r8j6
       TINKERBELL_GRPC_AUTHORITY: $TINKERBELL_HOST_IP:42113
       TINKERBELL_CERT_URL: http://$TINKERBELL_HOST_IP:42114/cert
       DATA_MODEL_VERSION: 1
+      TINK_WORKER_IMAGE: tinkerbell/tink/tink-worker:v0.6.0-eks-a-v0.0.0-dev-build.1627
+      USE_ABSOLUTE_ACTION_IMAGE_URI: "true"
```

Comment:
what does this do?

Reply:
suggest a better name for this, if you have it. This is basically saying that the action images in the tinkerbell action template have the full uri and will allow the bypassing of the hardcoded prepending of registry onto those. This way tink-worker and action images can reside in different registries. Action images themselves can each one come from a different registry potentially also. I have all this described in the PRs for hook/boots/tink-worker. Before I upstream those, I plan to have something explaining these.

Reply:
Do we namespace env variables? I see there's a bunch that aren't namespaced. I've had horrible experiences maintaining code bases that don't clearly name and namespace env vars.

Reply:
This is tinkerbell stack code. We are not going to use this as it is post-beta. But want to make this more convenient for beta, so minimally required changes here. 
Also, let's discuss during office hours about how exactly you talk about namespacing them. It might be good to submit those upstream to tinkerbell for other people's benefit for future use.